app/web/features/search/SearchPage.tsx

```diff
@@ -220,8 +243,8 @@ export default function SearchPage({
           setLocationResult={setLocationResult}
           lastActiveFilter={lastActiveFilter}
           setLastActiveFilter={setLastActiveFilter}
+          onChange={handleHostingStatusFilterChange}
           hostingStatusFilter={hostingStatusFilter}
-          setHostingStatusFilter={setHostingStatusFilter}
```

Comment:
We will handle this function in `SearchPage` since that's where the `hostingStatusFilter` state is, then just pass the function.